relationships.md

```diff
@@ -39,8 +39,8 @@ The leftId and rightId parameter in JSON may still be present, but they will be
   "_embedded": {
     "relationshipType": {
       "id": 1,
-      "leftLabel": "isAuthorOfPublication",
-      "rightLabel": "isPublicationOfAuthor",
+      "leftwardLabel": "isAuthorOfPublication",
+      "rightwardLabel": "isPublicationOfAuthor",
```

Comment:
To avoid any further confusion and to clarify others, I think we could swap, or reorder to:
```
"rightwardLabel": "isPublicationOfAuthor",
"leftwardLabel": "isAuthorOfPublication",

```      
Starting firstly with the rightward. In my mind, the relations always starts from the left item (Publication) to right item (Author) and this is expressed by the `rightwardLabel` . If in all produced documentation, regarding relations, the Atmire's picture and this order is showed, I think others will follow along easily.

Reply:
@paulo-graca would that be the only order you would want to be updated?
@tdonohue would you agree this helps make it easier to understand?

Reply:
@benbosman : I don't feel strongly enough either way here. I'm fine with it being reordered in the documentation. But, in the actual response it might appear in any order (as I don't think order matters there at all)